airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -643,6 +648,41 @@ def read_records(
             else:
                 raise e
 
+    def _validate_response(self, response: requests.Response, stream_state: Mapping[str, Any]) -> bool:
+        """
+        Ticket Audits endpoint doesn't allow filtering by date, but all data sorted by descending.
+        This method used to stop making requests once we receive a response with cursor value greater than actual cursor.
+        This action decreases sync time as we don't filter extra records in parse response.
+        """
+        data = response.json().get(self.response_list_name)
+        created_at = data[0].get(self.cursor_field)
+        cursor_date = (stream_state or {}).get(self.cursor_field) or self._start_date
+        return created_at > cursor_date
```

Comment:
Should it be `>=` to avoid missing any records?

Reply:
good point, will add it